### ISSUE TITLE
Can't log in to bloks.io anymore with Scatter 11

### ISSUE THREAD

**Issue by Soleone** [GetScatter/ScatterDesktop#433]
Both in Brave and in Chrome on Windows 10 it keeps spinning for me and the console shows these errors:

![image](https://user-images.githubusercontent.com/2923/61676441-edd3de80-acc9-11e9-8e75-4f364eaf51e5.png)

Funnily enough on the computer of my wife where Scatter regularly crashes (see my other issue from today) it works.

**Comment by Soleone** [comment]
Still happening. Another user in Telegram reported the same issue.

Wasn't there supposed to be a ScatterJS update that is supposed to fix something like this? Maybe I'm recalling it wrong.

![image](https://user-images.githubusercontent.com/2923/63142429-bcdc8600-bfb7-11e9-800c-e95bbecd97c0.png)

**Comment by Soleone** [comment]
Another report: https://t.me/Scatter/98683

**Comment by Soleone** [comment]
I just noticed something interesting, will see if other users can replicate:

When I go to Apps --> My Apps and find Bloks.io in the list and try to remove it, it actually doesn't work, it just stays, no error then in the console:

![image](https://user-images.githubusercontent.com/2923/63903500-bb5d8580-c9db-11e9-8b73-90bca5033738.png)

I have the feeling that the login issues on the website might be related to the permission stored in Scatter, maybe it is supposed to ask for authorization again to use the site, but something fails then and the dialog doesn't come up.

---

No idea if related but shortly after that I saw the following message in my console:

> internal/process/warning.js:23 (node:4468) MaxListenersExceededWarning:
> Possible EventEmitter memory leak detected. 11 seed listeners added. Use emitter.setMaxListeners() to increase limit

**Comment by sergchernata** [comment]
Same issue here. Is there any workaround?

**Comment by nsjames** [comment]
Does this also happen with Scatter 12?

**Comment by sergchernata** [comment]
Yes, it still happens. Remove button doesn't do anything. No errors in console.
